planet/cli/features.py

```diff
@@ -0,0 +1,184 @@
+from contextlib import asynccontextmanager
+import json
+import click
+from click.exceptions import ClickException
+
+from planet.cli.io import echo_json
+from planet.clients.features import FeaturesClient
+from planet.geojson import split_ref
+
+from .cmds import coro, translate_exceptions
+from .options import pretty, compact
+from .session import CliSession
+
+
+@asynccontextmanager
+async def features_client(ctx):
+    async with CliSession() as sess:
+        cl = FeaturesClient(sess, base_url=ctx.obj['BASE_URL'])
+        yield cl
+
+
+@click.group()  # type: ignore
+@click.pass_context
+@click.option('-u',
+              '--base-url',
+              default=None,
+              help='Assign custom base Features API URL.')
+def features(ctx, base_url):
+    """Commands for interacting with Features API"""
+    ctx.obj['BASE_URL'] = base_url
+
+
+@features.command()  # type: ignore
```

Comment:
not critical, but if we're applying the same annotations (command, pass_context, coro, etc.), these can be unified using a single convenience def, something like (where sequence_of_annotations is the "stack" of common ones)

```
def features_command(f):
  for ann in sequence_of_common_annotations:
    f  = ann(f)
  return f
```

then each command is reduced to the set of unique annotations, making it simpler to read...

Reply:
done!